misc/GNUmakefile

```diff
@@ -15,8 +15,6 @@ release: export
 
 	mv $(TEMP)/$(NGINX)/auto/configure $(TEMP)/$(NGINX)
 
-	mv $(TEMP)/$(NGINX)/docs/text/LICENSE $(TEMP)/$(NGINX)
-	mv $(TEMP)/$(NGINX)/docs/text/README $(TEMP)/$(NGINX)
```

Comment:
this change breaks make zip, source paths should be adjusted to move LICENCE and README files from $(TEMP)/$(NGINX)